src/pushNotifications/WasteReminder.ts

```diff
@@ -124,6 +129,7 @@ const deleteReminderSetting = async (id: number | string) => {
       })
       .catch((error) => {
         console.warn('An error occurred while deleting a reminder setting:', error);
+        serverConnectionAlert(false);
```

Comment:
this is not the case if there is no internet connection but when something fails. in this case the alert message is wrong. checking the internet connection should happen before making the request with giving the user the alert. if there is no internet connection, we do not need to try to make a request.